servers/partner-ci/builders/common.prepare.sh

```diff
@@ -20,22 +20,25 @@ fi
 [ -z $ISO_FILE ] && (echo "ISO_FILE variable is empty"; exit 1)
 
 
-if [[ $ISO_FILE == *"Mirantis"* ]]; then
+if [[ $ISO_FILE == *"Mirantis"* ]] || [[ "${FUEL_RELEASE_VERSION}" == "10" ]] ; then
   export FUEL_RELEASE=$(echo $ISO_FILE | cut -d- -f2 | tr -d '.iso')
-fi
-if [[ $ISO_FILE == *"custom"* ]]; then
+else
   export FUEL_RELEASE=90
 fi
 
-if [[ "${UPDATE_MASTER}" == "true" ]] && [[ ${FUEL_RELEASE} != *"80"* ]]; then
-  [ ${SNAPSHOTS_ID} ] && export SNAPSHOTS_ID=${SNAPSHOTS_ID} || export SNAPSHOTS_ID=${CUSTOM_VERSION:10}
-else
-  export SNAPSHOTS_ID="released"
+if [[ "${UPDATE_MASTER}" == "true" ]]; then
+  case "${FUEL_RELEASE}" in
+    *100* ) export SNAPSHOTS_ID=$(echo $ISO | cut -d- -f3 | tr -d '.iso') ;;
+    *90* ) export SNAPSHOTS_ID=${${SNAPSHOTS_ID}:=${CUSTOM_VERSION:10}} ;;
+    *80* ) echo "incompatible version for snapshots_id" ;;
+    *70* ) echo "incompatible version for snapshots_id" ;;
+    *61* ) echo "incompatible version for snapshots_id" ;;
+     *   ) export SNAPSHOTS_ID="released"
+  esac
 fi
 
 [ -z "${SNAPSHOTS_ID}" ] && { echo SNAPSHOTS_ID is empty; exit 1; }
```

Comment:
remove the check `[ -z "${SNAPSHOTS_ID}" ] && { echo SNAPSHOTS_ID is empty; exit 1; }`